src/elements/LinkButton/LinkButton.tsx

```diff
@@ -1,30 +1,62 @@
 import React from 'react';
 import { useNavigation } from 'react-navigation-hooks';
-import { View, Text } from 'react-native';
+import { View, Text, TouchableOpacity } from 'react-native';
+import * as colors from '@util/colors';
 import styles from './LinkButton.styles';
 
 interface LinkButtonProps {
 	text: string;
+	textColor?: string;
+	borderColor?: string;
+	disabled?: boolean;
 	destination?: string;
 	onPress?: (any) => void;
 }
 
-export default ({ text, destination, onPress }: LinkButtonProps) => {
+export default ({
+	text,
+	destination,
+	textColor = colors.NAVY_BLUE,
+	borderColor = colors.BANANA_YELLOW,
+	disabled = false,
+	onPress = () => {},
+}: LinkButtonProps) => {
 	const { navigate } = useNavigation();
 	const buttonFunction = destination
 		? () => navigate(destination)
 		: onPress && (func => onPress(func));
 
 	return (
 		<View style={styles.container}>
-			<View style={styles.textContainer}>
-				<Text
-					style={styles.text}
-					onPress={buttonFunction}
+			<TouchableOpacity
+				disabled={disabled}
+				activeOpacity={0.3}
+				onPress={buttonFunction}
+			>
+				<View
+					style={[
+						styles.textContainer,
+						{
+							borderColor: disabled && borderColor === colors.BANANA_YELLOW
```

Comment:
Logic is still incorrect in order to achieve the correct disabled styling.
> ![disabled-link-button-review](https://user-images.githubusercontent.com/9058130/81871285-58d9f480-952c-11ea-8a7d-0949c4641250.png)

1. The second button should have: text with disabled grey coloring.
2. The last button's border shouldn't change when disabled. No matter the state of the button, the border color should remain constant. Explore the Figma card in order to see how colors change between different button states. 

_Code to reproduce image_
```tsx
<LinkButton
	disabled={false}
	borderColor="pink"
	textColor="black"
	text="Enabled"
	onPress={() => { console.log('button clicked'); }}
/>

<LinkButton
	disabled={true}
	borderColor="pink"
	textColor="black"
	text="Disabled"
	onPress={() => { console.log('button clicked'); }}
/>

<LinkButton
	disabled={false}
	borderColor={BANANA_YELLOW}
	textColor={NAVY_BLUE}
	text="Enabled"
	onPress={() => { console.log('button clicked'); }}
/>

<LinkButton
	disabled={true}
	borderColor={BANANA_YELLOW}
	textColor={NAVY_BLUE}
	text="Disabled"
	onPress={() => { console.log('button clicked'); }}
/>
```